README.md

```diff
@@ -44,6 +44,23 @@ You have two ways of telling you ROS 2 application which XML to use:
 1. Placing your XML file in the running directory under the name `DEFAULT_FASTRTPS_PROFILES.xml`.
 2. Setting environment variable `FASTRTPS_DEFAULT_PROFILES_FILE` to your XML file.
 
+### Change publication mode
+
+Another way to change easily the publication mode without the need of defining a XML file is to use the environment variable `RMW_FASTRTPS_PUBLICATION_MODE`.
+This variable has lower precedence than `RMW_FASTRTPS_USE_QOS_FROM_XML`.
+Therefore, it is only taken into account when `RMW_FASTRTPS_USE_QOS_FROM_XML` is not set (or given a value different than `1`).
+The admissible values are:
+* `ASYNCHRONOUS`: asynchronous publication mode.
+Setting this mode implies that when the publisher invokes the write operation, the data is copied into a queue, a notification about the addition to the queue is performed, and control of the thread is returned to the user before the data is actually sent.
+A background thread (asynchronous thread) is in turn in charge of consuming the queue and sending the data to every matched reader.
+* `SYNCHRONOUS`: synchronous publication mode.
+Setting this mode implies that the data is sent directly within the context of the user thread.
+This entails that any blocking call occurring during the write operation would block the user thread, thus preventing the application with continuing its operation.
+It is important to note that this mode typically yields higher throughput rates at lower latencies, since the notification and context switching between threads is not present.
+* `AUTO`: let Fast DDS select the publication mode. This implies using the publication mode set in the XML file or, failing that, the default value set in Fast DDS (which currently is set to `SYNCHRONOUS`).
+
+By default, `rmw_fastrtps` assumes this variable to be `ASYNCHRONOUS`, which is the publication mode in use if this environment variable is not set.
```

Comment:
Do you really mean "assumes" here? That is, is there an implicit assumption that the value is ASYNCHRONOUS? Or, does this mean that rmw_fastrtps sets the value, by default, to ASYNCHRONOUS? I suppose the code will clarify, but the description should be clear.

Reply:
I think it must be "if `RMW_FASTRTPS_PUBLICATION_MODE` is not set, then `rmw_fastrtps_cpp` behaves as if it were set to `ASYNCHRONOUS`.

Reply:
Done in 2bb6f5fbb1e7f971ab941c944998d0ca20a23c88